index.html

```diff
@@ -18,61 +18,45 @@
            <div class="container">
                <header>
                    <h1>Create A Secure Account</h1>
-                   <p>Welcome to the future of savings and investments</p>
+                   <!-- <p>Welcome to the future of savings and investments</p> -->
                </header>
-           <form action="" method="" class="">
+           <form action="/index.html" method="post" id="signup">
          <!-- <h1>Create Account</h1> -->
         <div class="details">
             <label for="firstname"><b>First Name</b> </label>
```

Comment:
Please try to use STRONG tag instead of bold. It will be helpful for screen readers.